test/integration_test.exs

```diff
@@ -16,7 +16,7 @@ defmodule LogflareLogger.IntegrationTest do
     Application.put_env(:logflare_logger_backend, :api_key, @api_key)
     Application.put_env(:logflare_logger_backend, :source_id, @source)
     Application.put_env(:logflare_logger_backend, :level, :info)
-    Application.put_env(:logflare_logger_backend, :flush_interval, 500)
+    Application.put_env(:logflare_logger_backend, :flush_interval, 900)
```

Comment:
replacing Application env at runtime is antipattern and will affect other tests, would need to refactor this to to get_env at the start then use `on_exit/0` to reinstate the env value.

Reply:
Wanted to reduce the number of changes as much as I could to avoid breaking the tests more 😅

Will do that because yeah this can be the source of a lot of problems / flakyness